src/state/models/ui/shell.ts

```diff
@@ -224,13 +224,16 @@ export class ShellUiModel {
   activeLightbox: ProfileImageLightbox | ImagesLightbox | null = null
   isComposerActive = false
   composerOpts: ComposerOpts | undefined
+  tickEveryMinute = Date.now()
```

Comment:
I feel like @pfrazee had a better name when he mentioned this to me but it left my brain immediately :P 

Ideas? Figured maybe we'd have more than one interval in the future for seconds/hours, but maybe not necessary to be so specific in naming.

Reply:
Eh this works for now